app/src/main/java/org/wikipedia/page/PageActivity.java

```diff
@@ -699,15 +700,32 @@ public void onDestroy() {
     public void onActionModeStarted(ActionMode mode) {
         super.onActionModeStarted(mode);
         if (!isCabOpen() && mode.getTag() == null) {
-            Menu menu = mode.getMenu();
-            menu.clear();
-            mode.getMenuInflater().inflate(R.menu.menu_text_select, menu);
+            modifyMenu(mode);
             ViewUtil.setCloseButtonInActionMode(pageFragment.requireContext(), mode);
             pageFragment.onActionModeShown(mode);
         }
         currentActionModes.add(mode);
     }
 
+    private void modifyMenu(ActionMode mode) {
+        Menu menu = mode.getMenu();
+        ArrayList<MenuItem> menuItemsList = new ArrayList<>();
+
+        for (int i = 0; i < menu.size(); i++) {
+            String title = menu.getItem(i).getTitle().toString();
```

Comment:
We actually _do_ want to remove the "Search Wikipedia" item, because we're already in the Wikipedia app.  And we can reliably check for `R.string.search_hint` because it will match in all languages.

Reply:
Done